README.md

```diff
@@ -654,6 +654,18 @@ $ lerna bootstrap --scope "package-*" --ignore "package-util-*" --include-filter
 # package matched by "package-*"
 ```
 
+#### --only-updated
+
+When exectuting a script or command, only run the script or command on packages that have been updated since the last release.
```

Comment:
It'd be nice to clarify the semantics of what `updated` means. Perhaps just add to this sentence:

> A package is considered "updated" using the same rules as `lerna updated`.